lib/ansible/modules/messaging/rabbitmq_user.py

```diff
@@ -102,10 +102,10 @@
     choices: [ "yes", "no" ]
   state:
     description:
-      - Specify if user is to be added or removed
+      - Specify if user is to be added, removed or append permissions
```

Comment:
`Specify if user is to be added, removed or append permission. Note I(append) was added in Ansible 2.3.`